src/Stripe.net/Services/SubscriptionSchedules/SubscriptionSchedulePhaseItemTrialFreeOptions.cs

```diff
@@ -0,0 +1,7 @@
+// File generated from our OpenAPI spec
+namespace Stripe
+{
+    public class SubscriptionSchedulePhaseItemTrialFreeOptions : INestedOptions
```

Comment:
I haven't seen another empty nested options. It's unusual but fine I guess?

Reply:
This is same feedback as Node.

Reply:
Confirming this is expected.